Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: deploy-stack remove-stack build-nginx
+
+STACK_NAME ?= cfa_opencti
+COMPOSE_FILE?= docker-compose.yml
+NGINX_TAG?=0.0.1
```

Comment:
According to the latest changes I made on Nginx version "0.0.6" should be the latest. The changes were: I changed server names to point to investigate.

Reply:
The `NGINX?=0.0.1` is the default version tag, but we can override this by passing the latest image tag on `make build-nginx`

Reply:
Oh okay no problem. But just to be in the know, maybe we will add it somewhere in the readme doc. Since I did not commit my Nginx changes.